tasks/localstorage-master.yml

```diff
@@ -10,3 +10,58 @@
 
 - name: Apply local-storage storage class
   shell: oc apply -f "{{ openshift_ansible_tmp }}/storage-class.yaml"
+
+# Storage provisioner for local volumes
+# Needed to cleanup old volumes
+
+- name: Create local storage project
+  shell: oc new-project local-storage
+  failed_when: no
+
+- name: Switch to local-storage project
+  shell: oc project local-storage
```

Comment:
This is racy - I would recommend to use `-n local-storage` on the subsequent commands